Marsiple/App/Todos/TodosViewController.swift

```diff
@@ -21,6 +21,10 @@ class TodosViewController: UIViewController {
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    override func viewWillAppear(_ animated: Bool) {
+        reloadData()
```

Comment:
Rather than `forcing a reload` on every appearance, I would like a more precise implementation where we don't need to do a `Wild West reload` to get correct state. 

What I mean is, I would like you to implement concise callbacks for each data modifying event and sync it individiually with the table view. This requires more concise coding, but is worth it, as you fully and precisely have control over what's happening.